onediff_comfy_nodes/benchmarks/scripts/run_oneflow_case_ci.sh

```diff
@@ -28,3 +28,9 @@ python3 scripts/text_to_image.py \
     -w $WORKFLOW_DIR/ComfyUI_IPAdapter_plus/ipadapter_advanced.json \
     --baseline-dir $STANDARD_OUTPUT/test_ipa
 # --output-images \
+
+python3 scripts/text_to_image.py \
+    --comfy-port $COMFY_PORT \
+    -w $WORKFLOW_DIR/txt2img.json \
+    --ssim-threshold 0.7 \
+    --baseline-dir $STANDARD_OUTPUT/txt2img/imgs # --output-images
```

Comment:
INFO [2024-07-08 05:35:44] - Result directory: results/exp5
INFO [2024-07-08 05:35:48] - Connected to server: 127.0.0.1:8188
INFO [2024-07-08 05:35:48] - Testing workflows: ['resources/workflows/oneflow/txt2img.json']
INFO [2024-07-08 05:35:50] - Current VRAM used: 9.85 GB	Maximum VRAM used: 9.85 GB
INFO [2024-07-08 05:35:52] - Current VRAM used: 10.84 GB	Maximum VRAM used: 10.84 GB
INFO [2024-07-08 05:35:54] - Current VRAM used: 10.97 GB	Maximum VRAM used: 11.01 GB
INFO [2024-07-08 05:35:54] - Image Size - Height: 1024px, Width: 1024px
INFO [2024-07-08 05:35:55] - Saved image to: results/exp5/imgs/image_0.png
INFO [2024-07-08 05:35:55] - SSIM value with baseline: 0.9302242057145111
INFO [2024-07-08 05:35:55] - Workflow 0 E2E:  6.42 seconds
INFO [2024-07-08 05:35:56] - Current VRAM used: 10.14 GB	Maximum VRAM used: 11.01 GB
INFO [2024-07-08 05:35:58] - Current VRAM used: 12.86 GB	Maximum VRAM used: 12.86 GB
INFO [2024-07-08 05:36:00] - Current VRAM used: 12.86 GB	Maximum VRAM used: 12.86 GB
INFO [2024-07-08 05:36:01] - Image Size - Height: 1024px, Width: 1024px
INFO [2024-07-08 05:36:01] - Saved image to: results/exp5/imgs/image_1.png
INFO [2024-07-08 05:36:02] - SSIM value with baseline: 0.9411892814425838
INFO [2024-07-08 05:36:02] - Workflow 1 E2E:  5.64 seconds
INFO [2024-07-08 05:36:02] - Current VRAM used: 11.14 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:04] - Current VRAM used: 12.87 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:06] - Current VRAM used: 12.87 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:08] - Image Size - Height: 1024px, Width: 1024px
INFO [2024-07-08 05:36:08] - Saved image to: results/exp5/imgs/image_2.png
INFO [2024-07-08 05:36:08] - Current VRAM used: 12.02 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:08] - SSIM value with baseline: 0.9380964532921482
INFO [2024-07-08 05:36:08] - Workflow 2 E2E:  5.74 seconds
INFO [2024-07-08 05:36:10] - Current VRAM used: 7.23 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:12] - Image Size - Height: 768px, Width: 512px
INFO [2024-07-08 05:36:12] - Saved image to: results/exp5/imgs/image_3.png
INFO [2024-07-08 05:36:12] - Current VRAM used: 8.87 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:12] - SSIM value with baseline: 0.8832520419135389
INFO [2024-07-08 05:36:12] - Workflow 3 E2E:  3.72 seconds
INFO [2024-07-08 05:36:14] - Image Size - Height: 768px, Width: 512px
INFO [2024-07-08 05:36:14] - Saved image to: results/exp5/imgs/image_4.png
INFO [2024-07-08 05:36:14] - Current VRAM used: 10.47 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:15] - SSIM value with baseline: 0.9541622253007164
INFO [2024-07-08 05:36:15] - Workflow 4 E2E:  1.85 seconds
INFO [2024-07-08 05:36:16] - Current VRAM used: 8.39 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:18] - Current VRAM used: 10.47 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:18] - Image Size - Height: 768px, Width: 512px
INFO [2024-07-08 05:36:19] - Saved image to: results/exp5/imgs/image_5.png
INFO [2024-07-08 05:36:19] - SSIM value with baseline: 0.8022629809826531
INFO [2024-07-08 05:36:19] - Workflow 5 E2E:  3.96 seconds
INFO [2024-07-08 05:36:20] - Current VRAM used: 5.42 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:21] - Image Size - Height: 768px, Width: 512px
INFO [2024-07-08 05:36:21] - Saved image to: results/exp5/imgs/image_6.png
INFO [2024-07-08 05:36:21] - SSIM value with baseline: 0.9505677667259042
INFO [2024-07-08 05:36:21] - Workflow 6 E2E:  1.89 seconds
INFO [2024-07-08 05:36:23] - Current VRAM used: 5.42 GB	Maximum VRAM used: 13.03 GB
INFO [2024-07-08 05:36:23] - Image Size - Height: 768px, Width: 512px
INFO [2024-07-08 05:36:23] - Saved image to: results/exp5/imgs/image_7.png
INFO [2024-07-08 05:36:23] - SSIM value with baseline: 0.9591790934021153
INFO [2024-07-08 05:36:23] - Workflow 7 E2E:  1.87 seconds
INFO [2024-07-08 05:36:23] - Closing connection to ComfyClient